projects/plugins/wpcomsh/connection/class-atomic-storage-provider.php

```diff
@@ -47,11 +49,19 @@ public function get( $option_name ) {
 
 			switch ( $option_name ) {
 				case 'blog_token':
-					return $persistent_data->JETPACK_BLOG_TOKEN; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+					return empty( $persistent_data->JETPACK_BLOG_TOKEN ) ? false : $persistent_data->JETPACK_BLOG_TOKEN; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
 
 				case 'id':
-					$blog_id = $persistent_data->JETPACK_BLOG_ID; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
-					return $blog_id ? intval( $blog_id ) : null;
+					return intval( $persistent_data->JETPACK_BLOG_ID ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
Are we ok with changing return value from  `null` to `0` for empty blog ID here?